Source/Core/DolphinQt/GCMemcardManager.h

```diff
@@ -74,7 +79,7 @@ class GCMemcardManager : public QDialog
   std::array<QLabel*, SLOT_COUNT> m_slot_stat_label;
 
   int m_active_slot;
-  int m_current_frame;
+  u64 m_current_frame = 0;
```

Comment:
This one feels a bit odd being initialized inline here while everything else is initialized everywhere - do you think it's viable to also move other initializers here?